src/lib/db/segment-store.ts

```diff
@@ -240,23 +240,21 @@ export default class SegmentStore implements ISegmentStore {
             throw new NotFoundError('No row');
         }
 
-        const segment: ISegment = {
+        return {
             id: row.id,
             name: row.name,
             description: row.description,
             project: row.segment_project_id || undefined,
             constraints: row.constraints,
             createdBy: row.created_by,
             createdAt: row.created_at,
-            usedInProjects: row.used_in_projects
-                ? Number(row.used_in_projects)
-                : 0,
-            usedInFeatures: row.used_in_projects
-                ? Number(row.used_in_features)
-                : 0,
+            ...(row.used_in_projects && {
```

Comment:
why not 0 by default?

Reply:
The same mapper method is used when loading **single context field**, but we are using  this value only **in list view**, so it will confuse customers to have the count, but 0 values.